octobot/interface_factory.pxd

```diff
@@ -0,0 +1,27 @@
+# cython: language_level=3
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+
+cdef class InterfaceFactory:
+    cdef public object octobot
+    cdef object logger
+
+    cdef public list interface_list
```

Comment:
Do you prefer `interface_list` or `interfaces` ?

Reply:
I used interface_list to be consistent with the rest but interfaces is better in my opinion (no need for type in variable name), we can switch if you aggree

Reply:
Yes you are right in other interfaces there are some `_list`. We should change all of it in another PR.